### PR TITLE
Allow to specify the repository install path

### PR DESCRIPTION
**Pull request by tennashi**
I'm https://github.com/motemen/ghq user.
It manages all Git repositories with `ghq.root`.
So I want to manage Vim plugins in the same directory.

I added the `VOLTREPOPATH` environment variable that specify the path to install Vim plugins.

### CODE REVIEW

**Comment by tyru** [comment]
Thank you for PR.
I haven't checked this for details.
Please change `VOLTREPOPATH` to `VOLTREPOSPATH` (because I named it "repos" directory...)

File: README.md [hunk 1]
```diff
@@ -94,6 +95,11 @@ See [the profile feature](https://github.com/vim-volt/volt#switch-set-of-plugins
 You can change base directory of volt by `VOLTPATH` environment variable.
 This is `$HOME/volt` by default.
 
+### VOLTREPOPATH
+
+You can change base directory of Vim plugins by `VOLTREPOPATH` environment variable.
+This is `$HOME/volt/repo` by default.
```
Comment (tyru) [inline README.md]:
```suggestion
This is `$HOME/volt/repos` by default.
```

**Comment by tennashi** [comment]
Thank you for your review.
> Please change VOLTREPOPATH to VOLTREPOSPATH (because I named it "repos" directory...)

It fixed!

**Comment by tyru** [comment]
Can you change target branch to devel?

**Comment by tennashi** [comment]
I changed.

**Comment by tyru** [comment]
Thanks!

**Pull request merged**